x-pack/plugins/enterprise_search/public/applications/app_search/components/result_field_value/result_field_value.tsx

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
```

Comment:
I more or less left this file "as-is" when porting it over.

I mainly just resolved any TS errors then wrote tests.